websocket_server.go

```diff
@@ -369,11 +370,33 @@ func (n *pushServer) start(ctx context.Context) {
 						}
 					}
 
+					id, setID := session.currentPushConfigID()
 					switch session.encodingWrite {
 					case elemental.EncodingTypeMSGPACK:
-						session.send(dataMSGPACK)
+						data := dataMSGPACK
```

Comment:
ℹ️ You *must* create a copy otherwise you would be adding someone else's ID to all events that are being pushed out.

Reply:
all of this is deoptimizing a lot's of things. Can we please find another way?